src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java

```diff
@@ -243,315 +93,506 @@
 @GinModules(PortalGinModule.class)
 public interface PortalGinInjector extends Ginjector {
 
-	public BulkPresenterProxy getBulkPresenterProxy();
-	
-	public GlobalApplicationState getGlobalApplicationState();
-	
-	public PresenterProxy<HomePresenter, Home> getHomePresenter();
-	
-	public EntityPresenter getEntityPresenter();
-	
-	public ProjectsHomePresenter getProjectsHomePresenter();
-	
-	public LoginPresenter getLoginPresenter();
-	
-	public AuthenticationController getAuthenticationController();
-	
-	public PasswordResetPresenter getPasswordResetPresenter();
-	
-	public RegisterAccountPresenter getRegisterAccountPresenter();
+	BulkPresenterProxy getBulkPresenterProxy();
 
-	public ProfilePresenter getProfilePresenter();
+	GlobalApplicationState getGlobalApplicationState();
+
+	PresenterProxy<HomePresenter, Home> getHomePresenter();

```

Comment:
Why all the extra newlines?  Like the removal of implicit "public" in interface, but the large amount of cleanup-related changes should probably be in a separate PR.

Reply:
I undid the new lines and kept the removal of the public keywords.